code/game/machinery/vending.dm

```diff
@@ -674,6 +675,8 @@
 					src.status_error = 1
 				else
 					src.status_message = "Please swipe a card or insert cash to pay for the item."
+					if(no_criminals)
+						src.status_message += " We are legally required to ask if you are currently wanted by any law enforcement organizations. If so, please cancel the purchase, announce your location to local law enforcement and wait for them to collect you."
```

Comment:
```suggestion
						status_message += " We are legally required to ask if you are currently wanted by any law enforcement organizations. If so, please cancel the purchase, announce your location to local law enforcement and wait for them to collect you."
```

Reply:
good change

as long as you're poking around in here @EvenInDeathIStillServe could you remove all the "src." from this file?